src/components/Main.js

```diff
@@ -14,28 +14,48 @@ class Main extends Component {
       <Container fluid>
       <h1>{this.props.message}</h1>
       <Row>
-        <Col><BeastImage renderBeast={beast[0]}/></Col>
-        <Col><BeastImage renderBeast={beast[1]}/></Col>
-        <Col><BeastImage renderBeast={beast[2]}/></Col>
+        <BeastImage renderBeast={beast[0]}/>
+        <BeastImage renderBeast={beast[1]}/>
+        <BeastImage renderBeast={beast[2]}/>
       </Row>
       <Row>
-        <Col><BeastImage renderBeast={beast[3]}/></Col>
-        <Col><BeastImage renderBeast={beast[4]}/></Col>
-        <Col><BeastImage renderBeast={beast[5]}/></Col>
+        <BeastImage renderBeast={beast[3]}/>
+        <BeastImage renderBeast={beast[4]}/>
+        <BeastImage renderBeast={beast[5]}/>
```

Comment:
Great work on this!

Now that you have covered Map(), consider workin' that in, by mapping over the json data (`beastObjects`)👍